examples/networking/xdp/xdp_drop_count.py

```diff
@@ -0,0 +1,135 @@
+#/usr/bin/env python
+#
+# xdp_drop_count.py Drop incomming packets on XDP layer and count for which
+#                   protocol type
+#
+# C code adapted from Brenden Blanco as found in kernel:samples/bpf/xdp1_kern.c
+#
+# Copyright (c) 2016 Jan Rüth
+# Licensed under the Apache License, Version 2.0 (the "License")
```

Comment:
In other words, yes please drop the whole python file, the end result will be cleaner from a licensing point of view.